cookbook/src/pages/prerequisites.mdx

```diff
@@ -7,17 +7,18 @@ description: Prerequisites and supported software
 
 Node.js is a prerequisite for installing the Merative Social Program Managment UI Addon Development Environment and for developing and deploying your application pages.
 
-| Supported software | Version | Prerequisite minimum | Operating system restrictions |
-| :-------------- |  :----------- | :----------- | :----------- |
-| Node.js | 14 LTS (latest) | 14.16.0 LTS (Fermium) | No |
-| | 12 LTS (latest) | 12.13.0 LTS (Erbium) | No |
+| Supported software | Version         | Prerequisite minimum | Operating system restrictions |
+| :----------------- | :-------------- | :------------------- | :---------------------------- |
+| Node.js            | 20 LTS (latest) | 20.12.2 LTS          | No                            |
+|                    | 18 LTS (latest) | 18.20.2 LTS          | No                            |
 
 ## Merative Social Program Management
 
 | UI Addon Development Environment | Social Program Management | IBM Carbon   |
-| :--------------                  |  :-----------             | :----------- |
+| :------------------------------- | :------------------------ | :----------- |
 | 1.0.0                            | 8.0.1, 8.0.2, 8.0.3       | v10          |
 | 1.1.0                            | 8.0.1, 8.0.2, 8.0.3       | v10          |
+| 1.2.0                            | 8.0.1, 8.0.2, 8.0.3       | v11          |
```

Comment:
This is one for Elaine really I guess, but we probably want to state that this new release 1.2.0 (when released) is compatiable with the latest releases of Curam 8.1.0.0, 8.1.1.0 and 8.1.2.0 that have been released in the interim